core/commands/add.go

```diff
@@ -49,7 +49,7 @@ remains to be implemented.
 	},
 
 	Arguments: []cmds.Argument{
-		cmds.FileArg("path", true, true, "The path to a file to be added to IPFS").EnableRecursive().EnableStdin(),
```

Comment:
On Mon, Apr 27, 2015 at 05:14:07PM -0700, Juan Batiz-Benet wrote:
> … what is the problem being solved here? what is wrong by keeping it
> as is for now? (the cmds lib will get a big facelift at some point,
> so we can make sure to address this then)

I didn't realize ‘ipfs add’ could read from stdin until you told me it
could [1].  It seemed easy enough to update the command so the docs
would reflect that reality, and why have fixable errors in your docs?

But I agree that this is more of a UX issue than a functional issue,
so if the cmds lib looks to hairy I'll just leave it alone ;).

[1]: https://github.com/ipfs/go-ipfs/pull/1136#issuecomment-96844109